site/content/docs/4.3/components/accordion.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: docs
+title: Accordion
+description: Provides an accordion to collapse content vertically.
+group: components
+aliases:
```

Comment:
this isn't wrong, please remove it

Reply:
Or if you keep it, remove the redirects from the alerts page

Reply:
I remove the redirects from the alerts page and kept these ones.

Reply:
Thanks @XhmikosR